server/services/offendersService.js

```diff
@@ -1275,11 +1275,11 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
   }
 
   function inconsistentCategorisation(dbRecord, pnomisStatus) {
```

Comment:
unrelated slight adjustment to change treatment of 'I' records. Helpful for CAT-677 Unsentenced not appearing on recat todo list